### DISCUSSION TITLE
[Bug反馈] 使用【最新 main 分支开发调试构建】插件与AMLL Player有冲突

### DISCUSSION THREAD

**Discussion by ITManCHINA** [amll-dev/applemusic-like-lyrics#247] [Q&A]
AMLL插件使用的是[这个构建版本](https://github.com/Steve-xmh/applemusic-like-lyrics/releases/tag/main-nightly)
![1720292361012](https://github.com/Steve-xmh/applemusic-like-lyrics/assets/50987405/af3fb12a-2eba-4c53-ade5-b2dff56fd05a)
![1720292127631](https://github.com/Steve-xmh/applemusic-like-lyrics/assets/50987405/cabc50e9-03d0-43e5-a92f-7f9cec5fd540)
AMLL Player用的是截止发此讨论时的[最新版本](https://github.com/Steve-xmh/applemusic-like-lyrics/actions/runs/9725228960)

---

## Bug描述
开启插件内的歌词播放器后，网易云变灰且不可操控；AMLL Player也无法正常工作。

---

### Some words
@Xionghaizi001 你有头绪嘛，救一救啊🥲

**Comment by Deathbedkizz** [comment]
我也有这个问题，同样使用的是上述版本，在打开网易云音乐进入歌词界面后整个窗口直接变灰

**Reply by ITManCHINA** [reply]
> 我也有这个问题，同样使用的是上述版本，在打开网易云音乐进入歌词界面后整个窗口直接变灰

这个问题，进网易云后直接进betterncm的界面，再去AMLL设置里关闭AMLL插件的播放器开关就行
![1720509960023](https://github.com/Steve-xmh/applemusic-like-lyrics/assets/50987405/3b9f8d2b-9b19-44ff-9590-62c28cfa8f78)

**Reply by Deathbedkizz** [reply]
我使用这个版本的amll在betterncm的插件设置里面点击amll无法看到设置：（

**Comment by Xionghaizi001** [comment]
> AMLL插件使用的是[这个构建版本](https://github.com/Steve-xmh/applemusic-like-lyrics/releases/tag/main-nightly) 

你为啥不去艾特开发者😂
